pkgs/misc/emulators/atari800/default.nix

```diff
@@ -1,16 +1,20 @@
-{ stdenv, fetchurl
+{ stdenv, fetchFromGitHub, autoreconfHook
 , unzip, zlib, SDL, readline, libGLU, libGL, libX11 }:
 
 with stdenv.lib;
 stdenv.mkDerivation rec{
   pname = "atari800";
-  version = "4.0.0";
+  version = "4.2.0";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/atari800/atari800/${version}/${pname}-${version}.tar.gz";
-    sha256 = "1dcynsf8i52y7zyg62bkbhl3rdd22ss95zs2s9jm4y5jvn4vks88";
+  src = fetchFromGitHub {
+    owner = "atari800";
+    repo = "atari800";
+    rev = "ATARI800_4_2_0";
```

Comment:
can we parameterize this by the version number by replacing the periods with underscores?

Reply:
I was thinking about it! But I am not an expert on string substitution functions in Nix language yet.
Otherwise, it looks a good idea to me.

Reply:
```suggestion
    rev = "ATARI800_${replaceChars ["."] ["_"] version}";
```